batchATX_test.go

```diff
@@ -221,26 +224,13 @@ func testBatchATXInvalidAddendum(t testing.TB) {
 	}
 }
 
-// TestBatchATXInvalidAddendum tests validating Addendum must be Addenda05
-func TestBatchATXInvalidAddendum(t *testing.T) {
-	testBatchATXInvalidAddendum(t)
-}
-
-// BenchmarkBatchATXInvalidAddendum benchmarks validating Addendum must be Addenda05
-func BenchmarkBatchATXInvalidAddendum(b *testing.B) {
-	b.ReportAllocs()
-	for i := 0; i < b.N; i++ {
-		testBatchATXInvalidAddendum(b)
-	}
-}
-
 // testBatchATXInvalidAddenda validates Addendum must be Addenda05 with record type 7
 func testBatchATXInvalidAddenda(t testing.TB) {
 	mockBatch := NewBatchATX(mockBatchATXHeader())
 	mockBatch.AddEntry(mockATXEntryDetail())
 	addenda05 := mockAddenda05()
 	addenda05.recordType = "63"
-	mockBatch.GetEntries()[0].AddAddenda(addenda05)
+	//mockBatch.GetEntries()[0].AddAddenda(addenda05)
```

Comment:
Do these calls need changed to `.Addenda05 = addenda05`?

Reply:
Yep